app/ui/browser/views/navbar/location.jsx

```diff
@@ -28,6 +37,18 @@ const LOCATION_BAR_STYLE = Style.registerStyle({
   borderRadius: '2px',
 });
 
+const LOCATION_BAR_AUTOCOMPLETE_STYLE = Style.registerStyle({
+  position: 'absolute',
```

Comment:
This makes it not look like the mockup. The code would also be simpler if we just made the autocompletion list part of the chrome, and not some floating thing.

https://people.mozilla.org/%7Eshorlander/stuff/images/fawkes-explorations-06-05-search.png